package.json

```
@@ -165,7 +165,7 @@
   },
   "dependencies": {
     "@brave/brave-ui": "0.40.4",
-    "@brave/leo": "github:brave/leo#2d6f2b23ea86148846eb70536d02b44e2c0c6426",
+    "@brave/leo": "github:brave/leo#5574f30dfba6729d9a3bff9528722d56d34c888c",
```

@fallaciousreasoning, this SHA is for the icon color update.

```suggestion
    "@brave/leo": "github:brave/leo#033a457665879461b5492edc66e11ceaf80e949d",
```

---

https://github.com/brave/leo/commit/033a457665879461b5492edc66e11ceaf80e949d